### PR TITLE
Add New Port Streets of Claus

### PR DESCRIPTION
**Pull request by MadShmupper**
# New Port for { Streets of Claus }

## Game Information
- **Title**: { Streets of Claus }
- **URL**: { https://redscreensoft.com/ }

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [x] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

**Review by kloptops — CHANGES_REQUESTED** [review]
Just the port.json changes today. 😄

File: ports/streetsofclaus/port.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        "image": null,
+        "rtr": true,
+        "exp": false,
+        "runtime": null,
+        "reqs": [],
+        "arch": [
+            "armhf"
+        ]
+    },
+    "status": {
```
[the remark this answers is not in the document]
Comment (MadShmupper) [reply ports/streetsofclaus/port.json]: Done. Thank you 👾

File: ports/streetsofclaus/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+## Notes
+
+Huge thank you to the great people at [Red Screen Soft](https://www.redscreensoft.com/) for allowing PortMaster to distribute their free games.
+Another big thank you to all who helped in testing this port!
+
```
Comment (Cebion) [inline ports/streetsofclaus/README.md:5]: Missing source to the game page

File: ports/streetsofclaus/cover.png
Comment (Cebion) [inline ports/streetsofclaus/cover.png:1]: The cover is two squided screenshots together.
Please do either a screenshot or screenshot with overlay cover.

**Review by Cebion — CHANGES_REQUESTED** [review]

**Review by kloptops — DISMISSED** [review]
👍

**Review by kloptops — APPROVED** [review]
👍

**Review by Cebion — APPROVED** [review]

**Pull request merged**